wolfcrypt/test/test.c

```diff
@@ -4235,12 +4235,17 @@ int chacha_test(void)
     ChaCha dec;
     byte   cipher[128];
     byte   plain[128];
+    byte   cipher_big[1305] = {0};
```

Comment:
Let's only do the new ChaCha20 big test if not BENCH_EMBEDDED. Please add `#ifndef BENCH_EMBEDDED` around these new tests. You might also consider an `WOLFSSL_SMALL_STACK` option to malloc the big test buffers.